polyglot/piranha/README.md

```diff
@@ -229,6 +425,19 @@ The edges can be labelled as `Parent`, `Global` or even much finer scopes like `
 First decide, what scopes you need to capture, for instance, in Java we capture "Method" and "Class" scopes. Once, you decide the scopes construct scope query generators similar to [java-scope_config](/polyglot/piranha/src/cleanup_rules/java/scope_config.toml). Each scope query generator has two parts - (i) `matcher` is a tree-sitter query that matches the AST for the scope, and (ii) `generator` is a tree-sitter query with holes that is instantiated with the code snippets corresponding to tags when `matcher` is matched.
 
 
+## Piranha Arguments : 
```

Comment:
Wonder if this shouldn't be either closer to, or at least linked from, the places that mention Piranha arguments in the python or CLI interfaces.

Reply:
I Agree. This is unreferenced anywhere.